addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/handler/YeelightHandlerBase.java

```diff
@@ -125,12 +126,10 @@ public void channelLinked(ChannelUID channelUID) {
         logger.debug("ChannelLinked -> {}", channelUID.getId());
         super.channelLinked(channelUID);
 
-        if (channelUID.getId().equals(YeelightBindingConstants.CHANNEL_BRIGHTNESS)) {
-            Runnable task = () -> {
-                mDevice.queryStatus();
-            };
-            scheduler.schedule(task, 500, TimeUnit.MILLISECONDS);
-        }
+        Runnable task = () -> {
```

Comment:
Since the Brightness Channel got removed for some bulbs `mDevice.queryStatus();` is not caled for every device. Maybe this is important, so I removed the if statement.